tests/test_dataops.py

```diff
@@ -143,19 +143,17 @@ def test_compute_tangential_and_cross_components(modeling_data):
         # [-1010.889584349285, -69.9059242788237])*0.7*1.0e12*1.0002565513832675
     }
     expected_curve = {  # <<TO BE ADDED IN THE FUTURE>>
-        'angsep': np.array([]),
-        'cross_shear': np.array([]),
-        'tangential_shear': np.array([]),
-        'cross_DS': np.array([]),
-        'tangential_DS': np.array([]),
+        'angsep': np.array([0.002175111279323424171, 0.003723129781247932167]),
+        'cross_shear': np.array([0.277590689496438781, 0.639929479722048944]),
+        'tangential_shear': np.array([-0.23009434826803484841, -0.02214183783401518779]),
+        # DeltaSigma expected values for clmm.Cosmology(H0=70.0, Omega_dm0=0.275, Omega_b0=0.025)
```

Comment:
I used the cosmology (H0=70.0, Omega_dm0=0.275, Omega_b0=0.025) to get these values. In the flat geometry, there is  a comment in line 139 saying that it was used the values (H0=67.66, Omega_dm0=0.262, Omega_b0=0.049), but in line 251, we make tests for (H0=70.0, Omega_dm0=0.275, Omega_b0=0.025), so it confused me and i thought it would be best to use the same parameters as the ones in the test.

Reply:
@m-aguena